packages/office-ui-fabric-react/src/components/Icon/Icon.tsx

```diff
@@ -39,7 +62,7 @@ export const Icon = (props: IIconProps): JSX.Element => {
             classNames.root
           ) }
       >
-        <Image { ...props.imageProps as any } />
+        { iconFromCustomRendererFn ? iconFromCustomRendererFn : <Image { ...props.imageProps as any } /> }
```

Comment:
So I imagine there is a scenario where I want to register a custom renderer for a subset of the icons, but not all of them (like just custom rendering for SVGs from a sprite). I guess the solution there is to return null or from the custom render function when it doesn't handle it? Can you put a comment saying that?

Reply:
Yep, the solution is to return null and it will fallback.  Will add a comment